src/WebSocket/TransportManager.php

```diff
@@ -238,13 +243,22 @@ public function broadcast(array $payload)
         $subscribers = $this->subscriptions()->subscribers($subscription);
 
         Log::v(' ', '', 'Subscribers ['.$subscription.']:');
-        Log::v(' ', '', json_encode($subscribers));
 
         if (! $field || empty($subscribers)) {
             return;
         }
 
-        $event = $field->transform(array_get($payload, 'event'));
+        $event = array_get($payload, 'event');
+
+        foreach ($event as $key => $value) {
+            if (is_array($value) && array_keys($value) == ["class", "id", "relations", "connection"]){
```

Comment:
@TGNThump When trying to test this out, the `class`, `relations` and `connection` keys are missing on the `$payload` array passed into the `RedisBroadcaster`... do I need to do anything in the `Event` class that was fired to get this to serialize properly or are you doing something manual?

Reply:
The modified version of the SerializesModels trait (Nuwave\Lighthouse\Subscriptions\SerializesModels) needs to be on the event. I couldn't think of any other way to do this automatically without screwing with Laravel core code, so if you have any ideas, feel free to change it.